tests/product/base_product_case.py

```diff
@@ -190,23 +190,25 @@ def build_installer_in_docker(self):
         finally:
             installer_container.tear_down_containers()
 
-    def copy_dist_to_host(self, local_dist_dir, dest_host):
+    def copy_dist_to_host(self, local_dist_dir, dest_host, cluster=None):
+        if not cluster:
+            cluster = self.docker_cluster
         for dist_file in os.listdir(local_dist_dir):
             if fnmatch.fnmatch(dist_file, "prestoadmin-*.tar.bz2"):
-                self.docker_cluster.copy_to_host(
+                cluster.copy_to_host(
                     os.path.join(local_dist_dir, dist_file),
                     dest_host)
 
-    def install_presto_admin(self, host=None):
+    def install_presto_admin(self, cluster, host=None):
```

Comment:
This is needed in the case you'd like to install presto-admin on a cluster that is not one of the defaults (just regular centos or centos with presto; when those clusters get provisioned the resulting DockerCluster gets assigned to self.docker_cluster). Examples where this is needed include single node clusters, like the ones needed for the installer tests.